devices/gledopto.js

```diff
@@ -451,6 +451,13 @@ module.exports = [
         description: 'Zigbee 6W E26/E27 Bulb RGB+CCT',
         extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
     },
+    {
+        zigbeeModel: ['GL-G-004P'],
+        model: 'GL-G-004P',
+        vendor: 'Gledopto',
+        description: 'Zigbee 7W Garden Light Pro RGB+CCT',
+        extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device